src/lib.rs

```diff
@@ -106,8 +106,8 @@ mod worker;
 pub use crate::error::Error;
 
 pub use crate::proto::{
-    Client, Connection, DataSnapshot, FaktoryState, Job, JobBuilder, JobId, Reconnect,
-    ServerSnapshot, WorkerId,
+    Client, Connection, DataSnapshot, FaktoryState, Job, JobBuilder, JobId, MutationFilter,
+    MutationFilterBuilder, MutationTarget, Reconnect, ServerSnapshot, WorkerId,
 };
```

Comment:
This is starting to be a lot of top-level exports. How do you feel about starting to group some of them the way we've done with the `pub mod ent` below? Like `pub mod mutate` for example? Then we can also rename them to avoid the `Mutation` name prefix.